src/base-styles/helperClasses/links/pointer.css

```diff
@@ -1,3 +1,3 @@
 .pointer-events-none {
```

Comment:
based on the current naming convention, this should probably be `pointerEvents--none`